src/System.Drawing.Common/tests/mono/System.Drawing.Imaging/IconCodecTests.cs

```diff
@@ -0,0 +1,2131 @@
+                Assert.Equal(0, bmp.GetPixel(12, 36).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(12, 40).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(12, 44).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(16, 0).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 4).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(16, 8).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(16, 12).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 16).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 20).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 24).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 28).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(16, 32).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(16, 36).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(16, 40).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(16, 44).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(20, 0).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(20, 4).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(20, 8).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(20, 12).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(20, 16).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(20, 20).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(20, 24).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(20, 28).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(20, 32).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(20, 36).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(20, 40).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(20, 44).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(24, 0).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(24, 4).ToArgb());
+                Assert.Equal(-16777216, bmp.GetPixel(24, 8).ToArgb());
+                Assert.Equal(-1, bmp.GetPixel(24, 12).ToArgb());
+                Assert.Equal(0, bmp.GetPixel(24, 16).ToArgb());
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Bitmap48Data()
+        {
+            string sInFile = Helpers.GetTestBitmapPath("48x48_one_entry_1bit.ico");
+            using (Bitmap bmp = new Bitmap(sInFile))
+            {
+                BitmapData data = bmp.LockBits(new Rectangle(0, 0, bmp.Width, bmp.Height), ImageLockMode.ReadOnly, PixelFormat.Format24bppRgb);
+                try
+                {
+                    Assert.Equal(bmp.Height, data.Height);
+                    Assert.Equal(bmp.Width, data.Width);
+                    Assert.Equal(PixelFormat.Format24bppRgb, data.PixelFormat);
+                    Assert.Equal(48, data.Height);
+
+                    int size = data.Height * data.Stride;
```

Comment:
size not used. I wonder if you're doing this to cause a property in data to be initialized.